features/admin.applications.v1/pages/application-edit.tsx

```diff
@@ -506,6 +562,355 @@ const ApplicationEditPage: FunctionComponent<ApplicationEditPageInterface> = (
+                    <ol className="spaced-list">
+                        { formData["useClientIdAsSubClaimForAppTokens"] && (
+                            <li>
+                                {
+                                    t("applications:forms.inboundOIDC.sections"
+                                        + ".legacyApplicationTokens.fields."
+                                        + "useClientIdAsSubClaimForAppTokens.label")
+                                }
+                            </li>
+                        )
+                        }
+                        { formData["omitUsernameInIntrospectionRespForAppTokens"] && (
+                            <li>
+                                {
+                                    t("applications:forms.inboundOIDC.sections"
+                                        + ".legacyApplicationTokens.fields."
+                                        + "omitUsernameInIntrospectionRespForAppTokens.label")
+                                }
+                            </li>
+                        )
+                        }
+                    </ol>
+                </ConfirmationModal.Content>
+            </ConfirmationModal>
+        );
+    };
+
+    /**
+     * Handles banner content update action which prepares data.
+     */
+    const handleBannerCheckBoxUpdate = (formDataValues: any) => {
+        mutateApplicationInboundConfigs().then((response: any) => {
+
+            let values: any = response.data;
+
+            if (formDataValues.get("omitUsernameInIntrospectionRespForAppTokens")){
+                values = {
+                    ...values,
+                    omitUsernameInIntrospectionRespForAppTokens:
+                        formDataValues.get("omitUsernameInIntrospectionRespForAppTokens")?.length > 0
+                };
+            } else {
+                values = {
+                    ...values,
+                    omitUsernameInIntrospectionRespForAppTokens: omitUsernameInIntrospectionRespForAppTokens
+                };
+            }
+
+            if (formDataValues.get("useClientIdAsSubClaimForAppTokens")) {
+                values = {
+                    ...values,
+                    useClientIdAsSubClaimForAppTokens:
+                    formDataValues.get("useClientIdAsSubClaimForAppTokens")?.length > 0
+                };
+            } else {
+                values = {
+                    ...values,
+                    useClientIdAsSubClaimForAppTokens: useClientIdAsSubClaimForAppTokens
+                };
+            }
+
+            setFormdata({ ...values });
+            setShowConfirmationModal(true);
+        });
+    };
+
+    /**
+     * Handles the banner data update action.
+     */
+    const handleBannerCheckBoxUpdateConfirmation = async (): Promise<void> => {
+        setBannerUpdateLoading(true);
+
+        return updateAuthProtocolConfig<OIDCDataInterface>(application?.id, formData, SupportedAuthProtocolTypes.OIDC)
```

Comment:
why returning this function and the return type set as `void` ?

Reply:
ce3f7f0210aea52dca4e1340286cc5ffcb701a43